src/TurbulenceConvection/update_aux.jl

```diff
@@ -160,6 +160,7 @@ function update_aux!(
         aux_en.q_tot[k] =
             max(val1 * aux_gm.q_tot[k] - val2 * aux_bulk.q_tot[k], 0) #Yair - this is here to prevent negative QT
         aux_en.h_tot[k] = val1 * aux_gm.h_tot[k] - val2 * aux_bulk.h_tot[k]
+        aux_en.e_kin[k] = val1 * aux_gm.e_kin[k] - val2 * aux_bulk.e_kin[k]
```

Comment:
instead of computing e_kin in the env from the various velocities I am backing out its value form the updraft and grid mean , it is not the same but should be ok, its small anyways